cnf/ext/repositories.bnd

```diff
@@ -1,7 +1,7 @@
 -plugin: \
 	aQute.bnd.deployer.repository.FixedIndexedRepo;name=GOSS Core;locations=https://raw.githubusercontent.com/GridOptics/GOSS/master/cnf/releaserepo/index.xml,\
 	aQute.bnd.deployer.repository.FixedIndexedRepo; name=GOSS Dependencies;   locations=https://github.com/GridOptics/GOSS-Repository/raw/master/dependencies/index.xml.gz,\
-	aQute.bnd.deployer.repository.FixedIndexedRepo;name=CIM to GLM;locations=https://raw.githubusercontent.com/GRIDAPPSD/Powergrid-Models/develop/cnf/release/index.xml,\
+	aQute.bnd.deployer.repository.FixedIndexedRepo;name=CIMHub;locations=https://raw.githubusercontent.com/poorva1209/CIMHub/master/cnf/release/index.xml,\
```

Comment:
probably shouldn't be referencing your branch.

Reply:
ok..I will change that. If you are testing make sure to use my fork because that pull request is not yet merged with CIMHub.

Reply:
Updated. Just be aware that this will not work until pull request on CIMHub is merged.